data/pools/transactionPool.go

```diff
@@ -125,7 +125,7 @@ func MakeTransactionPool(ledger *ledger.Ledger, cfg config.Local, log logging.Lo
 	}
 	pool.cond.L = &pool.mu
 	pool.assemblyCond.L = &pool.assemblyMu
-	pool.recomputeBlockEvaluator(make(map[transactions.Txid]basics.Round), 0)
+	pool.recomputeBlockEvaluator(make(map[transactions.Txid]ledgercore.IncludedTransactions), 0)
```

Comment:
TIL you can use a `nil` map if it's just being used as an inclusion set. You do need to be more careful though, because it's still a panic if you try to add a value to it.